graphjet-core/src/test/java/com/twitter/graphjet/algorithms/filter/TweetTimeRangeFilterTest.java

```diff
@@ -0,0 +1,42 @@
+package com.twitter.graphjet.algorithms.filters;
+
+import com.twitter.graphjet.algorithms.filters.TweetTimeRangeFilter;
+import com.twitter.graphjet.stats.Counter;
+import com.twitter.graphjet.stats.StatsReceiver;
+import org.junit.Assert;
+import org.junit.Rule;
```

Comment:
I believe junit.Rule for this class isn't necessary as we're not using timeout or expectedException